src/Dapr.Actors/Runtime/ActorManager.cs

```diff
@@ -352,6 +352,20 @@ private async Task<T> DispatchInternalAsync<T>(ActorId actorId, ActorMethodConte
                 await actor.OnPreActorMethodAsyncInternal(actorMethodContext);
                 retval = await actorFunc.Invoke(actor, cancellationToken);
 
+                // Proxies using Remoting eat the exception, instead inspect their response.
+                if (retval is Tuple<string, byte[]> remotingResponse)
```

Comment:
I think it would be much better if we could use an excpetion to signal this rather than a `Tuple<string, byte[]>`. Right now many layers of the system are coupled to this represnetation rather than something obvious.